community/modules/scheduler/htcondor-pool-secrets/main.tf

```diff
@@ -93,12 +114,32 @@ resource "google_secret_manager_secret_version" "pool_password" {
 
 # this secret will be populated by the Central Manager
 resource "google_secret_manager_secret" "execute_point_idtoken" {
-  secret_id = "${var.deployment_name}-execute-point-idtoken"
+  secret_id = "${var.deployment_name}-execute-point-idtoken${local.auto}"
 
   labels = local.labels
 
   replication {
-    auto {}
+    dynamic "auto" {
```

Comment:
Non-blocking: is there any good way to avoid this code duplication?

Reply:
I think the best way to avoid duplication would be to have a for_each on the resource level with a hardcoded set:

```tf
resource "google_secret_manager_secret" "secret" {
  for_each = toset(["pool-password", "execute-point-idtoken"])

  secret_id = "${var.deployment_name}-${each.key}${local.auto}"
```